src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -409,7 +409,12 @@ export class AttachToDropdown extends SelectBox {
 			let currentKernelName = this.model.clientSession.kernel.name.toLowerCase();
 			let currentKernelSpec = find(this.model.specs.kernels, kernel => kernel.name && kernel.name.toLowerCase() === currentKernelName);
 			if (currentKernelSpec) {
-				kernelDisplayName = currentKernelSpec.display_name;
+				//KernelDisplay Name should be Kusto when connecting to Kusto connection
```

Comment:
Isn't currentKernelSpec one for Kusto? If so cant we have that kernelSpec's display_name be set to the desired value ( 'Kusto') so that the old code works here as before?

Reply:
Hi Arvind, currentKernelSpec will not have Kusto or any other kernel aliases that we would introduce as currentKernelSpec retrieves kernels from standardKernels via activeClientSession. Kusto is not set up as a standard kernel. This is why this condition on the line below is not Kusto specific, but Kernel Alias specific (for future extensibility). Once we know that the connection that we are connecting to has a notebookKernelAlias we would make sure that we send the correct kernelDispalyName (Kusto) otherwise it would send SQL and not properly load the loadAttachToDropdown and change kernel to Kusto.